README.md

```diff
@@ -1,15 +1,6 @@
-# radicle
+# Radicle
 
-> because 'tis the Origin of the Root... The Radicle is likewise called the seminal Root.
-> — Vallemont, *Curiosities of nature and art in husbandry and gardening* (1707)
-
-A LISP in the spirit of [Black](http://pllab.is.ocha.ac.jp/~asai/Black/) and
-other colors. Easily define and interact with shared and upgradable state
-machines.
-
-Radicle has a [webpage](http://radicle.xyz/), and a [Getting Started
-Guide](http://docs.radicle.xyz/en/latest/guide/GettingStarted.html) which
-contain a lot more information on `radicle`.
+A peer-to-peer stack for code collaboration
 
```

Comment:
I feel it might be beneficial to add the 3 value props here as well:
- peer to peer + blurb
- terminal first + blurb
- programmable + blurb

Reply:
agree

Reply:
@juliendonck: Can you come up with the copy and create a PR? In the meantime I’m going to merge it to avoid further confusion

Reply:
ok